runtime/src/tasks/fifo/tasks-fifo.c

```diff
@@ -721,7 +717,13 @@ void chpl_task_executeTasksInList(void** p_task_list_void) {
                            child_ptask->id,
                            child_ptask->is_executeOn);
 
-    (*task_to_run_fun)(child_ptask->arg);
+    if (child_ptask->countRunningTasks)
```

Comment:
It only increments the task counts if it was requested to do so (it's conditional) so you'd have to look at what creates these ptasks and which set countRunningTasks=true. I believe that is the same as it used to be (just that it used to be done in callTaskWrapper).